src/meta/src/stream/stream_manager.rs

```diff
@@ -918,6 +918,9 @@ mod tests {
     async fn test_drop_materialized_view() -> MetaResult<()> {
         let services = MockServices::start("127.0.0.1", 12334, false).await?;
 
+        // Wait for bootstrap_recovery to finish.
+        tokio::time::sleep(Duration::from_secs(2)).await;
```

Comment:
Can we call `BarrierScheduler::wait_for_next_barrier_to_collect` to make this deterministic?

Reply:
During recovery, the barrier scheduler will be marked as blocked and all commands will be aborted. It could still be possible that the bootstrap recovery fails the `FLUSH` command somehow. 🥵

Reply:
Then what about repeating this until success? Since this is test codes, any hacks look good to me. I just don't want this to be flaky anymore in the future. 😂

Reply:
Okay, that's also acceptable to me. I will change it.